packages/connect/src/connector/index.ts

```diff
@@ -66,6 +66,11 @@ function getSmoldotProviderPromise(): Promise<SmoldotExtensionAPI> | undefined {
   if (typeof document !== "object" || typeof CustomEvent !== "function") return
   const lightClientProvider = getSmoldotExtensionProviders()
     .filter((detail) =>
+      // Ah, so this is to additionally filter to using only using our substrate connect extension
```

Comment:
the connect package specifically integrates with the substrate connect extension. so if the user has mutliple wallets installed, we can't just take the first one. we need to find the one that is substrate connect.

Reply:
the connect package u can say is a subset of the discovery package where it is scoped to only the substrate-connect extension